package/base-files/files/lib/functions.sh

```diff
@@ -158,7 +158,10 @@ insert_modules() {
 		if [ -f /etc/modules.d/$m ]; then
 			sed 's/^[^#]/insmod &/' /etc/modules.d/$m | ash 2>&- || :
 		else
-			modprobe $m
+			# Since busybox modprobe returns 0 on loading non-existing
+			# and 255 on loading already loaded modules
+			# its return-code is not useful        
+			modprobe $m || true 
```

Comment:
hmm... I don't agree.
This code doesn't ignore modprobe's return code. I think, we should do so, since its bogus: 255 can be an error (if the module doesn't exists) or success if it's already loaded. I

Thus I don't like to make "255" explicit and go for ignore its return code at all.